src/ecs/rendering/events.rs

```diff
@@ -0,0 +1,9 @@
+//! Events of the rendering system
+
+use renderer::types::Window;
+
+/// Even to modify a window.
+pub struct WindowModifierEvent {
+    /// The closure that modifies the window
+    pub modify: fn(&mut Window) -> (),
```

Comment:
This should be of type Box<FnMut(&mut Window) -> ()>, just to grant better flexibility.

Reply:
So this isn't really an event but a more a message to the window which is necessary because `Window` is thread-local?

Reply:
Yes